src/app/components/profile/profile.component.ts

```diff
@@ -171,6 +171,30 @@ export class ProfileComponent implements OnInit {
           placeholder: 'Affiliated To',
           type: 'text',
           value: this.user['affiliation']
+        },
+        {
+          isRequired: false,
+          label: 'google_scholar_url',
+          name: 'update_google_scholar_url',
+          placeholder: 'Google Scholar Url',
+          type: 'url',
+          value: this.user['google_scholar_url']
+        },
+        {
+          isRequired: false,
+          label: 'github_url;',
```

Comment:
Please remove the semi-colon `;` from label, it will give error while making back-end request

Reply:
done